src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
```

Comment:
Can you make this repo public and not private please?

Reply:
Fixed!